src/main/java/com/aventstack/extentreports/ExtentTest.java

```diff
@@ -1201,7 +1201,16 @@ public Status getStatus() {
     public Test getModel() {
         return test;
     }
-    
+
+    /**
+     * Convenience method for setting the description of the internal test model
+     *
+     * @param desc The description of the test case in the logs
+     */
+    public void setDescription(String desc) {
+        test.setDescription(desc);
```

Comment:
I would suggest we add a test here to check this method actually updates the Description.